### DISCUSSION TITLE
windows1. 2.11 unable to identify Mac1 2.11 plot file[Bug]

### DISCUSSION THREAD

**Discussion by jatmi** [Chia-Network/chia-blockchain#10602] [Support]
### What happened?

windows1. 2.11 unable to identify Mac1 2.11 plot fil
![微信截图_20220301093915](https://user-images.githubusercontent.com/31263308/156091339-fa111eec-a69c-424a-9a80-56ea5d5aa0d3.png)
e

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by loppefaaret** [comment]
Are you able to browse the drive from windows, and see your plot files ? 
Do you have any errors reported in your `debug.log` file, located under your users home folder, in `~\.chia\mainnet\log\debug.log` ? 
specifically, reports from "harvester" are of interest.
